### PR TITLE
Try fix vale jobs

### PR DESCRIPTION
**Pull request by MatKuhr**

### CODE REVIEW

File: docs/js/guides/how-to-use-the-approuter.mdx [hunk 1]
```diff
@@ -25,6 +25,13 @@ Its responsibilities consist of dispatching requests to backend microservices, a
 The application router checks if a given request has a valid JSON Web Token (JWT) when accessing a target service.
 If the request contains a valid JWT, the application router forwards the request to the target service; if the request does not contain a valid JWT, the user must authenticate.
 As we can see in the diagramm below, we use an Identity Provider (IdP) to authenticate, the request is redirected to an IdP where a user gets authenticated and then redirected back to the application router for passing further according to its desired destination.
+1
+
+123
```
Comment (FrankEssenberger) [inline docs/js/guides/how-to-use-the-approuter.mdx:30]: ?

Comment (FrankEssenberger) [reply docs/js/guides/how-to-use-the-approuter.mdx:30]: 1,2,3,...

**Review by FrankEssenberger — APPROVED** [review]
once you are happy with the fix, go ahead an merge.

**Comment by MatKuhr** [comment]
Tried a couple of things to make the Vale action work properly and consistently on changed files, didn't work. Results were rather random, up to the point of two identical PRs having different build results.

**Pull request closed without merging**